cmd/version.go

```diff
@@ -30,7 +30,7 @@ func VersionCmd() *cobra.Command {
 			}
 
 			fmt.Println(Version)
-			fmt.Println(term.Yellow(version.UpdateNotice(Version, "gotocompany/meteor")))
+			fmt.Println(term.Yellow(version.UpdateNotice(Version, "goto/meteor")))
```

Comment:
have you build and test locally to check if the command working properly now?

Reply:
yes, panic no longer there

Reply:
i think the main issue is here
https://github.com/goto/meteor/blob/ce1ccc2fa05f24392c487de6211eb9f942e0c0fc/Makefile#L70-L71

need to change to this
```
build:
	go build -ldflags "-X ${NAME}/cmd.Version=${VERSION}" ${NAME}
```

Reply:
right, also updated the Makefile